nesta_ds_utils/viz/altair/saving.py

```diff
@@ -70,26 +74,32 @@ def _save_png(
     )
 
 
-def _save_html(fig: Chart, path: os.PathLike, name: str, scale_factor: int):
+def _save_html(
```

Comment:
Tiny tiny aesthetic comment so please feel free to ignore, but I noticed you separated out parameters into different lines for _save_svg and _save_png but haven't here.

Reply:
The pre-commit linting checks made these changes to separate lines if the line exceeds a (PEP8?) character count (should be ~80?)